contracts/invariant.ral

```diff
@@ -1,26 +1,6 @@
 struct InvariantConfig { admin: Address, mut protocolFee: Percentage }
 
-struct SwapResult {
-    nextSqrtPrice: SqrtPrice,
-    amountIn: TokenAmount,
-    amountOut: TokenAmount,
-    feeAmount: TokenAmount
-}
-
-struct CalculateSwapResult {
-    amountIn: TokenAmount,
-    amountOut: TokenAmount,
-    startSqrtPrice: SqrtPrice,
-    targetSqrtPrice: SqrtPrice,
-    fee: TokenAmount
-}
-
-struct QuoteResult {
-    amountIn: TokenAmount,
-    amountOut: TokenAmount,
-    targetSqrtPrice: SqrtPrice
-}
-
+const MAX_FEE_TIERS = 32
```

Comment:
Shouldn't the const be in `constants.ral` as well as other structs moved to `structures/` directory?